app/assets/stylesheets/application.scss

```diff
@@ -133,7 +150,28 @@ nav {
   height: 50px;
 }
 
-.navbar-nav .active {
-  background: none;
+.navbar-inverse .navbar-nav>.active>a, .navbar-inverse .navbar-nav>.active>a:hover, .navbar-inverse .navbar-nav>.active>a:focus{
+  background: transparent;
   font-size: 1.2em;
+}
+
+.navbar-inverse .navbar-nav > .open > a, .navbar-inverse .navbar-nav > .open > a:hover, .navbar-inverse .navbar-nav > .open > a:focus{
+  background: $active-dark;
+  height: 140px;
+  transition: 0.5s;
+  z-index: 2;
+}
+
+.navbar-inverse .navbar-nav .dropdown-toggle {
```

Comment:
Merge rule `.navbar-inverse .navbar-nav .dropdown-toggle` with rule on line 142<br>Selector should have depth of applicability no greater than 2, but was 3